pom.xml

```diff
@@ -41,7 +41,7 @@
 
   <properties>
     <upstream.version>2.25.2</upstream.version>
-    <apache-camel.features.tesb.version>2.25.2.tesb1</apache-camel.features.tesb.version>
+    <apache-camel.features.tesb.version>2.25.2.tesb2</apache-camel.features.tesb.version>
```

Comment:
no need of upgrade here as 7.4.1 has not been released yet

Reply:
patch number increment reverted.